html/semantics/forms/form-submission-0/resources/file-submission.py

```diff
@@ -1,12 +1,9 @@
-from six import PY3
-
 from wptserve.utils import isomorphic_decode
 
 def main(request, response):
     testinput = request.POST.first(b"testinput")
-    if PY3:
-        # The test asserts the string representation of this FieldStorage
-        # object, but unfortunately the value property has different types in
-        # Python 2 and 3. Unify them to native strings.
-        testinput.value = isomorphic_decode(testinput.value)
+    # The test asserts the string representation of this FieldStorage
```

Comment:
Comment no longer needed?

Reply:
Yeah, I'll remove it.

Reply:
@foolip it seems you forgot to remove the comment.

Reply:
Hmm, I definitely made the change, but maybe it's left sitting on my computer. Will fix!

Reply:
I went too far: https://github.com/web-platform-tests/wpt/pull/28758